pyseir/icu/infer_icu.py

```diff
@@ -0,0 +1,107 @@
+import json
```

Comment:
Couple things:

1. I'd like this to have the same structure as the rt, since they both independently support a single key metric.
2. I see the code to calculate the denominator also living here so we can deliver the complete metric in the api. So fast following with a couple more helper functions.
3. I'd like to keep the ICU Utilization code as separate from the PySEIR code, so it's clear what we would remove when we deprecate the SEIR v4 model. If anything, I'd like ICU estimator and Rt estimator to live on the same level as the pyseir directory. They aren't SEIR based models.